src/status.rs

```diff
@@ -431,6 +439,19 @@ impl StatusBuilder {
         self
     }
 
+    /// Add an extension to the `Status` object.
+    ///
+    /// The prefix `ext_` will automatically be prepended to the name, if not
+    /// already present.
+    pub fn add_extension<S: Into<String>>(mut self, name: S, value: Value) -> Self {
+        let mut key = name.into();
+        if key.starts_with("ext") {
```

Comment:
Actually, the thing you mentioned wasn't fixed yet. But I had inverted the logic :)